FueledUtils/ReactiveSwiftUIKit/ControlProtocol+Tapped.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2020, Fueled Digital Media, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ReactiveCocoa
+import ReactiveSwift
+
+private var tapActionStorage: UInt8 = 0
+private var tapActionKey: UInt8 = 0
+
+extension Reactive where Base: ControlProtocol {
+	///
+	/// The action to be triggered when the button is tapped.
+	/// This mirrors the `pressed` property native in `ReactiveCocoa`, but uses a
+	/// protocol to represents the button rather than hardcode it to classes,
+	/// allowing for any `UIControl` to use this method.
+	///
+	var tapped: TapAction<Base>? {
```

Comment:
I'd welcome a new name here, as `pressed` is already used, and they semantically are pretty much the same